fontc/src/main.rs

```diff
@@ -655,11 +701,15 @@ impl Workload {
 
                     debug!("Updating graph for new glyph {glyph_name}");
 
-                    self.jobs_pending
-                        .get_mut(&AnyWorkId::Be(BeWorkIdentifier::Glyf))
-                        .unwrap()
-                        .dependencies
-                        .insert(id.clone());
+                    // It would be lovely if our new glyph was in glyf and hmtx
+                    // loca hides with hmtx
```

Comment:
I'm confused by the sentence "loca hides with hmtx"... did you mean "glyf"?

Reply:
...yes

Reply:
Fixed